pkg/apis/operator/v1alpha1/const.go

```diff
@@ -37,6 +37,9 @@ const (
 	ClusterTasksParam      = "clusterTasks"
 	PipelineTemplatesParam = "pipelineTemplates"
 
+	// Hub Params
+	DevconsoleIntegrationParam = "enable-devconsole-integration"
```

Comment:
should we follow camel case similar to other param?
`EnableDevConsoleIntegration`

Reply:
fixed